### PR TITLE
Geojson routes

### PR DESCRIPTION
**Pull request by l-vincent-l**

### CODE REVIEW

**Comment by kinnou02** [comment]
probablement lié à: http://jira.canaltp.fr/browse/NAVITIAII-577

**Comment by antoine-de** [comment]
same here, rebase :recycle:

**Comment by l-vincent-l** [comment]
rebase done

**Comment by antoine-de** [comment]
jenkins, test this please

**Comment by l-vincent-l** [comment]
jenkins, test this please

File: source/type/type.proto [hunk 1]
```diff
@@ -151,6 +151,11 @@ message Line {
     repeated Code codes = 16;
 }
 
+message GeoJson {
+    required int32 length = 1;
```
Comment (TeXitoi) [inline source/type/type.proto]: Why not a floating point value ?

**Pull request merged**